### ISSUE TITLE
Install of BTC-RPC-Explorer failed

### ISSUE THREAD

**Issue by rootzoll** [raspiblitz/raspiblitz#3236]
On a test run installing from the WebUI the BTC-RPC-Explorer it falied and the backend API showed:
```
l 20 22:09:34 python[8011]: touch: cannot touch '/home/admin/btc-rpc-explorer.env': Permission denied
Jul 20 22:09:34 python[8011]: chmod: cannot access '/home/admin/btc-rpc-explorer.env': No such file or directory
Jul 20 22:09:34 python[8011]: ERROR:root:NO `result=` returned by script:
```

To be retested for final release.

**Comment by rootzoll** [comment]
Worked from SSH install script ... to be retested from WebUI (also on a fresh setup).

**Comment by rootzoll** [comment]
OK - got it fixed - closing issue for v1.8.0.
